src/backend/distributed/commands/utility_hook.c

```diff
@@ -553,7 +553,7 @@ static void
 ExecuteDistributedDDLJob(DDLJob *ddlJob)
 {
 	bool shouldSyncMetadata = ShouldSyncTableMetadata(ddlJob->targetRelationId);
-	int targetPoolSize = ddlJob->executeSequentially ? 1 : MaxAdaptiveExecutorPoolSize;
+	int targetPoolSize = MaxAdaptiveExecutorPoolSize;
```

Comment:
is it still worth keeping this as a parameter?

Reply:
Good point,  we can remove the parameter on `ExecuteUtilityTaskListWithoutResults`.